lib/News.ts

```diff
@@ -41,19 +45,49 @@ stay tuned for a Club Website event. Alternatively, you can view and contribute
 ];
 
 /**
- * Gets a news item, or null if it doesn't exist.
- * @param id - is the ID to look up the news item.
- */
-export async function getNewsItem(id: number): Promise<NewsItem | null> {
-    // TODO: Implement.
-    return tempNewsItems[id] || null;
+* Get news rows from database.
+* If no database connection has been made and process.env.NODE_ENV is not production
+* then provide dummy data so that it does not block development.
+* @param count - is the maximum number of entries to return.
+*/
+export async function getNews(count: number): Promise<NewsItem[]> {
+    try {
+        const rows = await pool.query("SELECT * FROM news LIMIT ?;", [count]);
+        rows.map(row => ({
+            date: row["date"].toString(),
+            title: row["title"],
+            description: row["description"],
+            author: row["author"],
+            content: row["content"],
+        }));
+        return rows;
```

Comment:
`rows.map` doesn't modify the array; it returns a new one. You should return the result from `rows.map`.